ground/gcs/src/plugins/config/configosdwidget.cpp

```diff
@@ -451,9 +451,7 @@ void ConfigOsdWidget::setupOsdPage(Ui::OsdPage * page, QWidget * page_widget, UA
     addUAVObjectToWidgetRelation(name, "BatteryChargeStatePosY", page->BatteryChargeStateY);
 
     // Center Mark
-    addUAVObjectToWidgetRelation(name, "CenterMark", page->CenterMarkEnabled);
-    page->CenterMarkEnabled->setProperty(trueString.toLatin1(), "Enabled");
-    page->CenterMarkEnabled->setProperty(falseString.toLatin1(), "Disabled");
+    addUAVObjectToWidgetRelation(name, "CenterMark", page->CenterMarkMode);
```

Comment:
Fair enough. I'd like to just move them all to the UI files rather than having a mixture. Can wait for a future cleanup.